source/installation/install-on-an-unattended-server/debian-server-installation/install-debian-server.txt

```diff
@@ -1,3 +1,116 @@
+.. _install-debian-server:
+
 =================
 Install On Debian
 =================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator on a Debian server. This installation method 
+allows Relational Migrator to run on an unattended server. By running on an 
+unattended server, Relational Migrator binds to any IP and port on the server 
+and is exposes it as a web application.
+
+Steps
+-----
+
+1. Download the latest ``.deb`` binary from the `release page <https://migrator-installer-repository.s3.ap-southeast-2.amazonaws.com/index.html>`__.
+
+#. Install Relational Migrator.
+
+   .. code-block:: bash
+
+      sudo apt install ./mongodb-relational-migrator_X.X.X-1_amd64.deb
+
+#. Change the directory to the Relational Migrator ``app/lib`` location.
+
+   .. code-block:: bash
+
+      cd /opt/mongodb-relational-migrator/lib/app/lib
+
+#. (Optional) To connect to Oracle or MySQL drivers, download and place the drivers 
+   into your Relational Migrator directory.
+
+   1. To download and install MySQL or Oracle drivers, see the following:
```

Comment:
(lists)
Please make the substeps alphabetical instead.

Reply:
Done.